### DISCUSSION TITLE
Make your work, work for you: Local Data and File Integration.

### DISCUSSION THREAD

**Discussion by zipo214** [athensresearch/athens#1349] [Feature Requests]
**Simple Concept Summary (TL;DR)** : Using Athens to create a personal back-linkable wiki-like experience to find and reference personal files, folders, pdfs, images and other local files would be a seriously powerful tool in the personal knowledge toolbox in our digital world.

**More info please:**
What if all of our documents and personal files on our computer could be dragged and dropped onto the UI of Athens, and it gave us a couple of options on what to do with that? Mainly, what if it asked us if we wanted to include it as a new <<_File-reference_ >> which works like a link (maybe more like a reddit markdown link where we link out but format/prettify/obscure the file name itself). Clicking on the link opens the file in its default program (or maybe in an Athens-friendly format). Another option Athens could give upon dragging and dropping the file is to open or create a _Project Block_ or _File Block_  which graphically shows and holds multiple documents where it previews the document in a scaled down manner akin to you were viewing the paper document on a desk. Allowing you to graphically visualize multiple documents related to one another. The file block could be organized by topic, or page and allows block references and backlinks to the title of the File Block or tags associated with it.  In order to minimize heavy system utilization, the Graphical visualization would be simplistic and static until you edit it or enter some form of "arrange and view" mode. Ultimately being able to reference local files in relation to your own daily notes or other Athens projects would deepen the efficiency of the program, and its effectiveness to minimize linking out or hunting down info outside the ecosystem. In conclusion, this feature would address other aspects of how to you manage a growing library of digitized knowledge-content the longer you acquire projects, essays, papers, code etc. It would help you find those files, reference them and backlink to them. All in attempts to better weave your knowledge together.

**P.S. (Additional context):**
    Being a local and private/secure application on the desktop is important, but the massive amounts of information and our aggregation of that information is becoming the norm and we need new methods to manage our personal knowledge, and I think Athens can use its infrastructure (when ready) to address this issue even deeper. 

Zettelkasten was devised to organize knowledge into referenced bite sized knowledge. For example, it would be useful in a scenario where you need to find your research from years before. You may have 5 to 15 different physical notebooks and you basically have no idea where you wrote down that one note about Descartes. You thought it was in your  _Intro to Philosophy_ notebook or the accompanying binder. It wasn't, in fact it was in your second year biology notebook because you were doing a project on neurobiology, and referenced Descartes because he had related thoughts and someone made the side-comment in neurobiology about "I think, therefore I am". Zettelkasten helps reference your knowledge. Athens will does the same. Life is good; but it can be better!

**Comment by jsmorabito** [comment]
sounds similar to the recently released feature on obsidian "You can now choose to show all files, even if Obsidian doesn’t recognize the file extension. This will allow you to link to those files, see them in the File Explorer, and open them from the Quick Switcher." [🔗](https://forum.obsidian.md/t/obsidian-release-v0-10-10/11966)

i agree would be very useful!
